lib/prosopite/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Prosopite
-  VERSION = "1.3.1"
+  VERSION = "0.0.1"
```

Comment:
[P2]  
I wonder if it's possible to use a custom versioning scheme (e.g. `0.0.1-holistics` to avoid confusion :thinking:

Reply:
Done [here](https://github.com/holistics/prosopite/pull/1/commits/41e92bda22c2472f0c8324643983ed0edae11c5c)